phoenicis-javafx/src/test/java/org/phoenicis/javafx/AppConfigurationInitialisationTest.java

```diff
@@ -16,8 +16,17 @@
  */
 public class AppConfigurationInitialisationTest {
 
+    public static void setupHeadless() {
```

Comment:
Please add some JavaDoc explaining what the method is responsible for and what the different options you set do.

Reply:
Done.